test_runner/performance/test_physical_replication.py

```diff
@@ -237,7 +237,7 @@ def test_replication_start_stop(
             ],
             env=master_env,
         )
-        replica_pgbench: list[Optional[subprocess.Popen[Any]]] = [None for _ in range(num_replicas)]
+        replica_pgbench: list[subprocess.Popen[Any] | None] = [None for _ in range(num_replicas)]
```

Comment:
NIT:

```python3
replica_pgbench: list[subprocess.Popen[Any] | None] = [None] * num_replicas
```

Reply:
This should be AnyStr instead of Any

Reply:
`AnyStr` is deprecated in 3.13 and will be removed in 3.18:
https://docs.python.org/3/library/typing.html#typing.AnyStr